src/targetSelector.ts

```diff
@@ -76,6 +76,10 @@ export class TargetSelector {
 
     this.selectedTarget = target;
 
+    // Memorize the selected device in the settings
+    const conf = vscode.workspace.getConfiguration("ledgerDevTools");
+    conf.update("defaultDevice", target, vscode.ConfigurationTarget.Global);
```

Comment:
As discussed, this should probably be saved in a new setting with the `WorkspaceFolder` scope.

Reply:
As discussed, we'll keep the storage in the local `.vscode/`, in the repo directory